doc/release-notes/8578-support-for-iso-639-3-languages.md

```diff
@@ -0,0 +1,17 @@
+The Controlled Vocabulary Values list for the metadata field Language in the Citation block has now been extended to include roughly 7920 ISO 639-3 values.
+
```

Comment:
Perhaps could/should list any entries in the old table that are now split into multiples so people can check for those and decide if they need updating.

Reply:
I've scanned through the other macrolanguages and it looks like the Pulaar and Pular entries are the only ones affected/where the name would change. We do have other cases where a macro language was in the old list and there are now individual language entries that could be more appropriate. Perhaps just add something like "Some of the language entries in the pre-v.6.4 list correspond to "macrolanguages" in ISO-639-3 and admins/users may wish to update to use the corresponding individual language entries from ISO-639-3. As these cases are expected to be rare (they do not involve major world languages), finding them is not covered in the release notes. Anyone who desires help in this area is encouraged to reach out to the Dataverse community via any of the standard communication channels."

Reply:
Added to release notes